module/Application/view/partial/main-nav.phtml

```diff
@@ -173,6 +173,7 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
                             <li><a href="<?= $this->url('activity_calendar') ?>"> <?= $this->translate('Option
                             calendar') ?></a></li>
                         <?php endif; ?>
+                        <li><a href="<?= $this->url('activity/career') ?>"> <?= $this->translate('Career related') ?></a></li>
```

Comment:
I agree, the current position is pretty unnoticeable and another position would probably become cluttered.
How about adding the link in the 'title', like this?
![image](https://cloud.githubusercontent.com/assets/8017652/24949140/e106fd50-1f6c-11e7-93fe-13429555dc0e.png)